clang/lib/DPCT/APINamesLIBCU.inc

```diff
@@ -0,0 +1,47 @@
+//===-------------------- APINamesLIBCU.inc --------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// Base API
+
+CALL_FACTORY_ENTRY("cuda::std::atomic_thread_fence", CALL(MapNames::getClNamespace() + "atomic_fence",
```

Comment:
I am not familiar with the atomic, but does this rewriter also cover the cuda::atomic_thread_fence?